packages/manager/modules/pci/src/projects/project/instances/instances.html

```diff
@@ -221,6 +222,14 @@
                     data-translate="pci_projects_project_instances_add_label"
                 ></span>
             </a>
+            <button
+                type="button"
+                class="ml-2 float-right oui-button oui-button_secondary oui-button_icon-only oui-button_small-width"
+                aria-label="{{:: 'pci_projects_project_instances_refresh' | translate }}"
+                data-ng-click="$ctrl.refreshInstances()"
+            >
+                <span class="fa fa-refresh" aria-hidden="true"></span>
```

Comment:
Isn't there a `oui-icon` for this ?

Reply:
there is one, but the icon appears very very mall. Its the extra thin icon